### PR TITLE
Fixing merge issue with passing TagFilter option to deployVisualStudio test agent

### PR DESCRIPTION
**Pull request by pryenugu**
Fixing merge issue with passing TagFilter option to deployVisualStudio test agent

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1 [hunk 1]
```diff
@@ -67,7 +67,7 @@ if ( [string]::IsNullOrEmpty($personalAccessToken))
...
+    Invoke-DeployTestAgent -TagFilter $testMachines -UserName $machineUserName -Password $machinePassword -PowerShellPort 5985 -TestMachineGroup $testMachineGroup -RunAsProcess $runAsProcess -LogonAutomatically $logonAutomatically -DisableScreenSaver $disableScreenSaver -AgentLocation $agentLocation -UpdateTestAgent $updateTestAgent -InstallAgentScriptLocation $installAgentScriptLocation -ConfigureTestAgentScriptLocation $configureTestAgentScriptLocation -CheckAgentInstallationScriptLocation $checkAgentInstallationScriptLocation -downloadTestAgentScriptLocation $downloadTestAgentScriptLocation -Connection $connection -PersonalAccessToken $personalAccessToken -DataCollectionOnly $isDataCollectionOnly
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: does this mean the param in cmd let has changed? what happens when agent is old and task is new? wont it fail. IMO we shuld be changing the existing param names we shipped in RTM

**Comment by pankajkhanzode** [comment]
Signing off.

File: Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1 (hunk 1 above)
Comment (pryenugu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: We have not changed any old param names, it is the same. It is just another non-mandatory parameter which comes into play only if he chooses to deploy using tags, so no compat issue.

Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: thanks. Didn't notice the if condition. that will only hit when if condition will be true. which will be when new version of task is being used. looks fine

**Pull request merged**